miner/block_builder.go

```diff
@@ -179,13 +177,13 @@ func (t *BlockBuilder) createBlock(id types.LayerID, atxID types.AtxId, eligibil
 		ATXs: atx,
 		Txs:  txs,
 	}
-	atxs := " "
+	var atxs []string
 	for _, x := range b.ATXs {
-		atxs += "," + x.ShortId()
+		atxs = append(atxs, x.ShortId())
 	}
 
-	t.Log.Info("I've created a block in layer %v. id: %v, num of transactions: %v, votes: %d, viewEdges: %d atx %v, atxs:%v",
-		b.LayerIndex, b.Id, len(b.Txs), len(b.BlockVotes), len(b.ViewEdges), b.ATXID.String()[:5], atxs)
+	t.Log.Info("I've created a block in layer %v. id: %v, num of transactions: %v, votes: %d, viewEdges: %d, atx: %v, atxs: %v",
```

Comment:
please verify regex in automation tests matched the modified string

Reply:
Thanks for pointing that out! 😅

Looks like we only use `I've created a block in layer` to find this, and this hasn't changed. I guess this would have also broken the automation test, and it passes.